cluster/gce/gci/configure-helper.sh

```diff
@@ -1195,8 +1195,10 @@ function start-kube-addons {
     setup-addon-manifests "addons" "dns"
     local -r dns_controller_file="${dst_dir}/dns/kubedns-controller.yaml"
     local -r dns_svc_file="${dst_dir}/dns/kubedns-svc.yaml"
+    local -r dns_sa_file="${dst_dir}/dns/kubedns-sa.yaml"
```

Comment:
I think you don't need this line `local -r dns_sa_file="${dst_dir}/dns/kubedns-sa.yaml"`, because `setup-addon-manifests "addons" "dns"` already copy all the files.

And it seems like "kube-addon-manager.yaml" is not copied on to the master, which means start_kube_addons() met some errors in the middle.